### PR TITLE
Add Supply chain whitepaper directory structure and assessment

### PR DESCRIPTION
**Pull request by rjulian**
Additionally added relevant README to whitepaper directory, update top level README to reflect existence of whitepaper.

Will add whitepaper markdown once out of draft status.

### CODE REVIEW

File: supply-chain-security/README.md [hunk 1]
```diff
@@ -20,4 +20,10 @@ There is on-going work to establish best practices in this area. The list of
 in the [catalog of supply chain compromises](./compromises) suggests some
 mitigation techniques for the more well understood categories.
 
+# Supply Chain Security Whitepaper
 
+SIG-Security has put work into a comprehensive software supply chain whitepaper
```
Comment (TheFoxAtWork) [inline supply-chain-security/README.md:25]: STAG (Security Technical Advisory Group)

File: supply-chain-security/README.md [hunk 2]
```diff
@@ -20,4 +20,10 @@ There is on-going work to establish best practices in this area. The list of
 in the [catalog of supply chain compromises](./compromises) suggests some
 mitigation techniques for the more well understood categories.
 
+# Supply Chain Security Whitepaper
```
Comment (TheFoxAtWork) [inline supply-chain-security/README.md:23]:
```suggestion
# Supply Chain Security Paper
```

File: supply-chain-security/README.md [hunk 3]
```diff
@@ -20,4 +20,10 @@ There is on-going work to establish best practices in this area. The list of
...
 
+SIG-Security has put work into a comprehensive software supply chain whitepaper
+highlighting best practices for high and medium risk environments. Please check 
+out [the whitepaper](./supply-chain-security-whitepaper/software-supply-chain-security-whitepaper.md) and corollary [secure supply chain assessment document](./supply-chain-security-whitepaper/secure-supply-chain-assessment.md) to learn more.
```
Comment (TheFoxAtWork) [inline supply-chain-security/README.md:27]:
```suggestion
out [the paper](./supply-chain-security-whitepaper/software-supply-chain-security-whitepaper.md) and corollary [secure supply chain assessment document](./supply-chain-security-whitepaper/secure-supply-chain-assessment.md) to learn more. 
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
+Software Supply Chain Security Whitepaper
+==========================================
+
+# About
+
+The Software Supply Chain Security Whitepaper is a SIG-Security effort
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/README.md:6]:
```suggestion
The Software Supply Chain Security Paper is a STAG effort 
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+# About
+
+The Software Supply Chain Security Whitepaper is a SIG-Security effort 
+to ensure the cloud native community has access to information about building, 
+distributing, deploying, and running secure software supply chains.
+
+## Updates to the paper 
+
+The Software Supply Chain Security Whitepaper (SSCSWP) is intended to be a living
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/README.md:12]:
```suggestion
The Software Supply Chain Security Paper (SSCSWP) is intended to be a living 
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+The Software Supply Chain Security Whitepaper is a SIG-Security effort 
+to ensure the cloud native community has access to information about building, 
+distributing, deploying, and running secure software supply chains.
+
+## Updates to the paper 
+
+The Software Supply Chain Security Whitepaper (SSCSWP) is intended to be a living 
+document created and maintained for the community, by its members.  
+
+Updates to the whitepaper, suggestions for updates, or discussion for updates
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/README.md:15]:
```suggestion
Updates to the paper, suggestions for updates, or discussion for updates
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+should initiate with an [issue](https://github.com/cncf/sig-security/issues) submitted to the repo and labeled with
+"suggestion" and "whitepaper".
+
+### Markdown 
+
+The living SSCSWP is maintained in [markdown](https://github.com/cncf/sig-security/blob/master/supply-chain-security/software-supply-chain-security-whitepaper.md) and all updates will be made in
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/README.md:21]:
```suggestion
The living SSCSP is maintained in [markdown](https://github.com/cncf/sig-security/blob/master/supply-chain-security/software-supply-chain-security-whitepaper.md) and all updates will be made in
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+ markdown.  
+
+### Contributing updates 
+
+All members of the community are welcome to contribute updates to the SSCSWP.
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/README.md:26]:
```suggestion
All members of the community are welcome to contribute updates to the SSCSP.
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+We ask potential contributors to refer to the original design decisions, 
+listed below, as guidance when determining the content of their updates.
+
+It is highly recommended that you seek peer review for your updates beyond that
+of the Technical Leads and Co-Chairs of the SIG.
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/README.md:31]:
```suggestion
of the Technical Leads and Co-Chairs of the STAG.
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+Once the PR is submitted, please place the link in the CNCF SIG-Security Channel
+for the SSCSWP: [#sig-security-supply-chain-wg](https://cloud-native.slack.com/archives/C01KL0B4LKC) to request a review.
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/README.md:34]:
```suggestion
for the SSCSP: [#sig-security-supply-chain-wg](https://cloud-native.slack.com/archives/C01KL0B4LKC) to request a review.
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+
+### Versioning and publishing 
+
+It is expected that many minor updates will occur, corrections to grammar, 
+spelling, clarification in language, translations, etc.  When these occur 
+they are considered minor changes to the overall content and will not warrant
+ the regeneration of the PDF.
+
+When significant changes to the intent, content, or numerous minor changes
+occur, the SSCSWP working group will assess and determine if a new major version
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/README.md:44]:
```suggestion
occur, the SSCSP working group will assess and determine if a new major version
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+of the PDF needs published.  When this decision is made, the markdown content
+will be converted to text document and sent to the CNCF technical writers to
+create the PDF.  The PDF will then be published back into the repository
+annotating the new version, updating the links in the README.md accordingly.
+
+Minor updates to the markdown shall receive a minor version bump indicated in the
+Metadata table of the document and recorded as WIP.  When enough significant
+changes have been recorded, the markdown will be placed "In Review" (via PR) and
+solicited to the CNCF SIG-Security and TOC mailing list for review, at a minimum.
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/README.md:53]:
```suggestion
solicited to the CNCF STAG and TOC mailing list for review, at a minimum.
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+
+Upon completion of review, the SIG-Security TOC Liaison shall provide final
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/README.md:55]:
```suggestion
Upon completion of review, the STAG TOC Liaison shall provide final
```

**Review by TheFoxAtWork — CHANGES_REQUESTED** [review]
Need to change some acronyms to be more inclusive as well as support the SIG > STAG rename.  I think we are ready for the full conversion to MD once @lizrice finalizes her comments and we adjudicate.

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+distributing, deploying, and running secure software supply chains.
+
+## Updates to the paper 
+
+The Software Supply Chain Security Paper (SSCSP) is intended to be a living 
+document created and maintained for the community, by its members.  
+
+Updates to the paper, suggestions for updates, or discussion for updates
+should initiate with an [issue](https://github.com/cncf/sig-security/issues) submitted to the repo and labeled with
+"suggestion" and "whitepaper".
```
[the remark this answers is not in the document]
Comment (TheFoxAtWork) [reply supply-chain-security/supply-chain-security-whitepaper/README.md:17]: Agreed.  Call it "supply chain" @lumjjb

File: supply-chain-security/README.md [hunk 15]
```diff
@@ -20,4 +20,10 @@ There is on-going work to establish best practices in this area. The list of
 in the [catalog of supply chain compromises](./compromises) suggests some
 mitigation techniques for the more well understood categories.
 
+# Supply Chain Security Paper
```
Comment (TheFoxAtWork) [inline supply-chain-security/README.md:23]:
```suggestion
# Supply chain security paper
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 16]
```diff
@@ -0,0 +1,62 @@
+Software Supply Chain Security Whitepaper
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/README.md:1]:
```suggestion
Software Supply Chain Security Paper
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md (hunk 14 above)
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/README.md:17]:
```suggestion
"suggestion" and "supplychain".
```

File: supply-chain-security/supply-chain-security-whitepaper/README.md [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
+
+### Versioning and publishing 
+
+It is expected that many minor updates will occur, corrections to grammar, 
+spelling, clarification in language, translations, etc.  When these occur 
+they are considered minor changes to the overall content and will not warrant
+ the regeneration of the PDF.
+
+When significant changes to the intent, content, or numerous minor changes
+occur, the SSCSP working group will assess and determine if a new major version
```
[the remark this answers is not in the document]
Comment (TheFoxAtWork) [reply supply-chain-security/supply-chain-security-whitepaper/README.md:44]: I think governance docs are a larger discussion for papers going forward and should definitely be added.  #621

File: supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md [hunk 18]
```diff
@@ -0,0 +1,70 @@
+# Evaluating your Supply Chain Security
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md:1]:
```suggestion
# Evaluating your supply chain security
```

Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md]:
```suggestion
# Evaluating your supply chain security
```

File: supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md [hunk 19]
```diff
@@ -0,0 +1,69 @@
+# Evaluating your supply chain security
+
+## A Framework for Supply Chain Evaluation
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md:3]:
```suggestion
## A framework for supply chain evaluation
```

File: supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+
+So how does your supply chain stack up? Here are some questions to ask
+yourself:
+
+### Verify Source Code
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md:8]:
```suggestion
### Verify source code
```

File: supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+
+-   Do you enforce review and approval of contributions prior to merging?
+
+-   Are branch protection rules in place?
+
+-   Do you enforce MFA and SSH keys for human-entities? Do you have a plan in place for rotating SSH keys at regular intervals or following a key leak?
+
+-   Do you limit the access of automation agents (like CI/CD pipelines) following the principles of least privilege and just-in-time?
+
+### Verify Materials
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md:29]:
```suggestion
### Verify materials
```

File: supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md [hunk 22]
```diff
@@ -0,0 +1,69 @@
...
+
+-   Do you automatically perform Software Composition Analysis on dependencies when they are downloaded/installed?
+
+-   Do you monitor dependencies for updates and security issues?
+
+-   Do you build dependencies yourself instead of relying on public package managers?
+
+-   Do you create an SBOM of your own artefacts?
+
+### Protecting Build Pipelines
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md:43]:
```suggestion
### Protecting build pipelines
```

File: supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md [hunk 23]
```diff
@@ -0,0 +1,69 @@
...
+
+-   Do you use separate build workers/containers for each step in your build pipeline?
+
+-   Do your pipeline orchestrator pass in the environments, tools, and commands allowed on each build worker?
+
+-   Do you network isolate your build workers and pipeline as much as possible?
+
+-   Do you produce verifiable, reproducible builds?
+
+### Protecting Artefacts and Deployments
```
Comment (TheFoxAtWork) [inline supply-chain-security/supply-chain-security-whitepaper/secure-supply-chain-assessment.md:63]:
```suggestion
### Protecting artefacts and deployments
```

**Review by TheFoxAtWork — APPROVED** [review]
LGTM @lumjjb please double check and if nothing found in err please merge in so we can get the paper added to the repo in the correct location.

**Comment by lumjjb** [comment]
Sorry, but teeny tiny minor nit, could we rename the directory from `whitepaper` to `paper`.

**Pull request merged**